src/utils/uploadCsv.ts

```diff
@@ -0,0 +1,22 @@
+import {apiUrl} from "../config/api";
+import {saveCsv} from "./saveCsv";
+
+export const uploadCsvFile = async (csvFile: string) => {
+    const formData = new FormData()
+    const fileBlob = new Blob([csvFile], {type: 'text/csv'})
+    formData.append('csvFile', fileBlob, 'csvFile.csv');
+    try {
+        const response = await fetch(`${apiUrl}/admin/validate-csv`, {
+            method: 'POST',
+            body: formData,
+        })
+        if (!response.ok) {
```

Comment:
tutaj można będzie dodać komunikat jak ja w swoim tasku ze snakbarem. Można osobne zdanie na to zrobić i przydzielić komuś kto ma mniej czasu